mdoc/src/main/scala/mdoc/internal/markdown/Markdown.scala

```diff
@@ -151,10 +157,12 @@ object Markdown {
     val lifted = f.lift
     val clazz = ev.runtimeClass.asInstanceOf[Class[T]]
     class Madness {
-      val visitor = new NodeVisitor(new VisitHandler[T](clazz, visit))
-      def visit(e: T): Unit = {
-        lifted.apply(e)
-        visitor.visitChildren(e)
+      val visitor = new NodeVisitor(new VisitHandler[T](clazz, new MadnessVisitor))
+      class MadnessVisitor extends Visitor[T] {
```

Comment:
Not sure why, but this stopped compiling on 2.11, I made the conversion here more explicit.